variational_rate_of_aging_autoencoder.py

```diff
@@ -0,0 +1,214 @@
+import numpy as np
+from multiphenotype_utils import (get_continuous_features_as_matrix, add_id, remove_id_and_get_mat, 
+    partition_dataframe_into_binary_and_continuous, divide_idxs_into_batches)
+import pandas as pd
+import tensorflow as tf
+from dimreducer import DimReducer
+
+from general_autoencoder import GeneralAutoencoder
+from standard_autoencoder import StandardAutoencoder
+from variational_autoencoder import VariationalAutoencoder
+
+class VariationalRateOfAgingAutoencoder(VariationalAutoencoder):
+    """
+    Implements a variational rate-of-aging autoencoder.
+    Generative model: 
+    rate_of_aging ~ some_prior (currently, rate_of_aging = exp(scaling_factor * N(0, I)), so this is a log-normal prior centered around 1). 
```

Comment:
why exp(1/2)? not sure i understand this.

Reply:
The normal is centered around 0 but the exp of that is centered around half, I think? Because mean of exp of normal is exp(mu + sigma^2/2) or something like that?

Just have to be careful when you're talking about log-normals: are you referring to the normal part or after exp.